packages/flutter_tools/lib/src/project_validator.dart

```diff
@@ -11,7 +13,116 @@ abstract class ProjectValidator {
   /// Can return more than one result in case a file/command have a lot of info to share to the user
   Future<List<ProjectValidatorResult>> start(FlutterProject project);
   /// new ProjectValidators should be added here for the ValidateProjectCommand to run
-  static const List <ProjectValidator> allProjectValidators = <ProjectValidator>[
-    // TODO(jasguerrero): add validators
+  static List <ProjectValidator> allProjectValidators = <ProjectValidator>[
+    GeneralInfoProjectValidator(),
   ];
 }
+
+class GeneralInfoProjectValidator extends ProjectValidator{
```

Comment:
Can you add a dartdoc?

Reply:
These actually need to be `///` for them to be parsed by the dartdoc tool: https://dart.dev/guides/language/effective-dart/documentation#do-use--doc-comments-to-document-members-and-types

Reply:
Also, this should be of the format: single line summary sentence, followed by empty line, then remainder: https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph